packages/react-ui-core/src/Form/Field.js

```diff
@@ -199,6 +205,7 @@ export default class Field extends Component {
         key: `label-${inputId}`,
         htmlFor: inputId,
         className: theme.Label,
+        ...(ignoreAction) && { 'data-tag_action': 'ignore' },
```

Comment:
does this work for the negative? asking bc i legitimately don't know what happens when you spread `...(false)`

should this be a ternary instead `...(ignoreAction ?{ 'data-tag_action': 'ignore' } : {})`

Reply:
The falsy value should short circuit and wouldn't return the KVP.

See this fiddle for an example: https://jsfiddle.net/vnoktp2u/

I think another way of writing this could be a ternary.  I could go either way, I use lots of ternaries so honestly I was just trying something different here.

Reply:
I supposed the closing brackets should, or could, follow the closing curly bracket.  Seems to work either way in the fiddle above.